config/settings/base.py

```diff
@@ -67,3 +77,39 @@
 
 
 STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {
+    # Use hyperlinked styles by default.
+    'DEFAULT_MODEL_SERIALIZER_CLASS':
+        'rest_framework.serializers.ModelSerializer',
+    'DEFAULT_PERMISSION_CLASSES': (
+        'roles.permissions.MFLPermission',
+    ),
+    'DEFAULT_FILTER_BACKENDS': (
+        'rest_framework.filters.DjangoFilterBackend',
```

Comment:
Add the `OrderingFilter` backend here too...there was a comment to that effect in the first review.

See: http://www.django-rest-framework.org/api-guide/filtering/#orderingfilter